runtime/legion/legion_views.cc

```diff
@@ -545,7 +549,9 @@ namespace Legion {
                                                    const unsigned index,
                                                    const AddressSpaceID source,
                            LegionMap<ApEvent,FieldMask>::aligned &preconditions,
-                             std::set<RtEvent> &applied_events, bool can_filter)
+                                              std::set<RtEvent> &applied_events,
+                                                   bool tracing,
```

Comment:
Why are we just passing a bool in all these cases and not the 'PhysicalTraceInfo' pointer (if there is one)?